google/cloud/bigquery/dataset.py

```diff
@@ -501,7 +501,9 @@ def from_api_repr(cls, resource: dict) -> "AccessEntry":
         if len(entry) != 0:
             raise ValueError("Entry has unexpected keys remaining.", entry)
 
-        return cls(role, entity_type, entity_id)
+        config = cls(role, entity_type, entity_id)
+        config._properties = copy.deepcopy(resource)
+        return config
```

Comment:
Could you please explain why this change is necessary?

Reply:
@Linchin  this function should not be changed. Thanks for catching it. I reverted it back.